velox/type/DecimalUtil.h

```diff
@@ -249,6 +245,22 @@ class DecimalUtil {
     }
   }
 
+  inline static std::optional<int128_t> computeValidSum(
```

Comment:
Maybe add some comments for this function. Looks overflow as `0`, `1`, `-1` has some special meaning.